pkg/protodep/README.md

```diff
@@ -0,0 +1,53 @@
+# protodep
```

Comment:
nit: might be worth extracting the todos/forward-looking comments to a separate section at the end. When reading this I'd like to know how to use the tool as it stands now